pom.xml

```diff
@@ -553,6 +553,7 @@
                                 <generateClientAsBean>false</generateClientAsBean>
```

Comment:
In `AbfrageService` den auskommentierten Code entfernen.

![image](https://github.com/it-at-m/isi-backend/assets/56300801/369d995a-de02-408a-b75c-41318dcb81b6)